### PR TITLE
build: create packages for deb and rpm

### PR DESCRIPTION
**Pull request by kgarner7**
Support building deb/rpm/arch packages, with autoinstall scripts for setting up navidrome. 
Defaults:
- `/etc/navidrome/navidrome/toml` configuration file
- `/var/lib/navidrome` data directory
- `/opt/navidrome/music` music directory

Note that for rpm, you need `ffpmeg-free` (which appears to be available in fedora) or ffmpeg (third-party provider)

Not sure if a post-remove script is also desired. Could at least add a notice "You should delete this user"?

### CODE REVIEW

**Comment by deluan** [comment]
Thanks! As we discussed in Discord, let's wait for #3125 to be merged, so we can try to consolidate the _service_ installation.

File: contrib/navidrome.service [hunk 1]
```diff
@@ -11,15 +11,13 @@ WantedBy=multi-user.target
 User=navidrome
 Group=navidrome
 Type=simple
-ExecStart=/usr/bin/navidrome
+ExecStart=/usr/bin/navidrome --configfile "/etc/navidrome/navidrome.toml"
```
Comment (mintsoft) [inline contrib/navidrome.service:14]: Should be `navidrome service execute --configfile ...` now

Comment (deluan) [reply contrib/navidrome.service:14]: The idea is to not use this scripts, but use `navidrome service install` to auto-generate them.

Also, the syntax above should still work for Linux and macOS, right?

Comment (mintsoft) [reply contrib/navidrome.service:14]: Yeah it should, but I think it makes sense for us to prefer to run inside the Kardianos wrapper so we have consistent behaviour between OS's?

**Comment by mintsoft** [comment]
> `/etc/navidrome/navidrome/toml` configuration file

Is that a typo? I'd expect this to be at `/etc/navidrome/navidrome.toml` or `/etc/navidrome.toml`

**Comment by kgarner7** [comment]
> > `/etc/navidrome/navidrome/toml` configuration file
> 
> Is that a typo? I'd expect this to be at `/etc/navidrome/navidrome.toml` or `/etc/navidrome.toml`

In the comment? Yes. It's .toml in the code

**Pull request merged**